app/views/eventDetails/paymentOptions.html

```diff
@@ -6,17 +6,12 @@ <h3 class="panel-title" translate>Credit Card</h3>
   </div>
   <div class="panel-body">
     <div class="form-group">
-      <div class="col-sm-12 stacked-spacing-col-sm" ng-if="originalConference.paymentGatewayType === 'AUTHORIZE_NET'">
-        <label translate>Select A Payment Gateway Type:</label>
-        <select class="form-control" ng-options="key as value.name for (key, value) in paymentGateways" ng-model="conference.paymentGatewayType"></select>
+      <div class="col-sm-12 stacked-spacing-col-sm">
+        <label translate>Payment Gateway Type Used:  {{conference.paymentGatewayType !== null ? conference.paymentGatewayType : 'Other (not TSYS)'}}</label>
```

Comment:
I'm wondering if it is useful to tell the user if their event is using TSYS or not. @hlbraddock do users interact with TSYS at all when requesting a `Merchant Account ID` or managing payments or is that intended to be a black box to the user?